media-fonts/inconsolata-lgc/inconsolata-lgc-1.1.0_p20140917.ebuild

```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit font
+
+DESCRIPTION="Modified version of Inconsolata Hellenic adding the Cyrillic alphabet"
+HOMEPAGE="https://github.com/DeLaGuardo/Inconsolata-LGC"
+SRC_URI="https://github.com/DeLaGuardo/Inconsolata-LGC/tarball/d086a618248121d61e3f3de64b4301230d1c860c -> ${P}.tar.gz"
```

Comment:
Could you move that long-ish hash to a separate variable (above DESCRIPTION)? Preferably name it `EGIT_COMMIT` to match convention used by git-r3.